python/ffsim/operators/fermi_hubbard.py

```diff
@@ -0,0 +1,87 @@
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fermi-Hubbard model Hamiltonian."""
+
+from ffsim._lib import FermionOperator
+from ffsim.operators.fermion_action import cre_a, cre_b, des_a, des_b
+
+
+def fermi_hubbard_1d(
+    norb: int,
+    tunneling: float,
+    interaction: float,
+    chemical_potential: float = 0,
```

Comment:
```suggestion
    interaction: float,
    *,
    chemical_potential: float = 0,
```
Let's force keyword arguments here so we can add more later without breaking the API.

Reply:
That's a good idea! I have added this now